src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -101,7 +114,8 @@ const WorkspaceTabs = ({
         callbacks: { onClone, onShare, onLock, onDelete, onLeave },
         workspaceInfo: { canShare, isAzureWorkspace, isLocked, isOwner, workspaceLoaded: !!workspace }
       })
-    ])
+    ]),
+    workspace && isAzureWorkspace && AzureWarning(isAzureWorkspace)
```

Comment:
starting with `workspace &&` helped fix an annoying visual effect: a brief flash of the banner while the workspace was being loaded (since `isAzureWorkspace` is , before disappearing.